lib/topology/logging.py

```diff
@@ -45,6 +45,29 @@
 """Simple map with the default logging levels."""
 
 
+class FileHandler(logging.FileHandler):
```

Comment:
Is it strictly necessary for the name of this class to be the same as the one in ```logging```?

Reply:
Probably not, changing name